helpdesk_mgmt_maintenance/models/maintenance_equipment.py

```diff
@@ -0,0 +1,41 @@
+# © 2023 Solvos Consultoría Informática (<http://www.solvos.es>)
+# License LGPL-3 - See http://www.gnu.org/licenses/lgpl-3.0.html
+
+from odoo import fields, models
+
+class MaintenanceEquipment(models.Model):
+    _inherit = "maintenance.equipment"
+
+    ticket_ids = fields.Many2many('helpdesk.ticket', string="Tickets")
```

Comment:
This works fine, but we should code it in a more secure way. Please look a this example

https://github.com/OCA/purchase-workflow/blob/338dbfa7768b76c6674400fbdffcf59f35f98873/purchase_request/models/purchase_order.py#L103-L110

and

https://github.com/OCA/purchase-workflow/blob/338dbfa7768b76c6674400fbdffcf59f35f98873/purchase_request/models/purchase_request_line.py#L97-L105

, in order to explicity link `ticket_ids` here and `equipment_ids` in ticket model. This is because other addons could create some alternative `Many2many` relations between these two models, that can cause conflict.

Moreover, both fields should be marked as `copy=False`, because e.g. when a ticket and/or an equipment are duplicated, relations shouldn't be kept.

Anb `ticket_ids` should be `readonly` IMO (not tested, I don't know if it can be a source of problems).